.github/workflows/spellcheck.yml

```diff
@@ -222,11 +233,13 @@ jobs:
 #W:fuzzer
```

Comment:
(Github isn't letting place this comment on the correct line). "fiddlings" isn't actually a real word.